### DISCUSSION TITLE
Add the ability to connect a validation rule for a URL

### DISCUSSION THREAD

**Discussion by qertis** [quasarframework/quasar#15662] [Ideas / Proposals]
The file [patterns.js](https://github.com/quasarframework/quasar/blob/dev/ui/src/utils/patterns.js) does not contain a RegEx for URLs. I suggest adding one so that URLs can be validated, for example, like this:

```
<q-select :rules="['url']" />
```

**Comment by metalsadman** [comment]
you can use your own regex

**Reply by qertis** [reply]
> you can use your own regex

I can, but I want to implement RFC 3986 for URL in the standard library, just like Quasar already implemented RFC 5322 for email
